python/python/bystro/ancestry/dummy_ancestry_response.py

```diff
@@ -1,150 +0,0 @@
-"""Provide dummy ancestry response for integration testing."""
```

Comment:
This file was unused and redundant with the test_ancestry_listener.py examples on how to build an AncestryJobCompleteMessage